arangosh/Benchmark/BenchFeature.cpp

```diff
@@ -286,14 +291,22 @@ void BenchFeature::start() {
         operationsCounter.incompleteFailures(),
         requestTime,
     });
+    histogrammStr = "Percentile: 85%    90%     95%      99%\n";
```

Comment:
it looks like the headlines are not properly aligned with the printed values later on.
Can we align them properly?

Reply:
ok, will do.